modules/extensions-support/src/main/java/org/mule/runtime/module/extension/api/loader/java/type/ExtensionParameter.java

```diff
@@ -134,6 +135,14 @@ default boolean isRequired() {
     return java.util.Optional.empty();
   }
 
+  /**
+   * @return Whether this parameter is considered content.
+   */
+  default boolean isContent() {
```

Comment:
this shouldn't be done here nor it should depend on the extension code but the extension model